### PR TITLE
forse ci siamo

### PR DESCRIPTION
**Pull request by Liber-R**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Data-Science-Project
 Project for the final examination of the second module of the course Computational Management of Data. It will be a great journey!
-Pietro :)
\ No newline at end of file
+Pietro :) romolooooooooooooooooooo
```
Comment (Liber-R) [inline README.md:3]: terribile

File: README.md
Comment (Pepe066) [inline README.md:1]: aaaaaaaa

File: README.md (hunk 1 above)
Comment (Pepe066) [inline README.md:3]: aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa

Comment (Liber-R) [reply README.md:3]: ok

**Pull request merged**